app/src/screens/ProposalDetail.js

```diff
@@ -22,7 +23,7 @@ import {
 } from '../components/ConvictionVisuals'
 import { addressesEqualNoSum as addressesEqual } from '../lib/web3-utils'
 
-const Wrapper = styled.div`
```

Comment:
This component is not used anymore right? Could it be deleted, please?

Reply:
Resolve in the next PR